{
  "totalNumberSkippingTests": 62,
  "step-durations": {
    "CheckoutBuggyBuild": 3,
    "ComputeClasspath": 2,
    "CloneRepository": 31,
    "ResolveDependency": 14,
    "TestProject": 684,
    "BuildProject": 353,
    "GatherTestInformation": 0,
    "PushIncriminatedBuild": 7,
    "SquashRepository": 10
  },
  "totalMemory": 450887680,
  "nbLibraries": 209,
  "bugType": "failing_passing",
  "workspace": "./workspace",
  "totalNumberErroringTests": 0,
  "repo": "druid-io/druid",
  "freeMemory": 340535248,
  "failingModule": "/root/workspace/druid-io/druid/226126724/server",
  "buildid": 226126724,
  "hostname": "repairnator",
  "totalNumberFailingTests": 4,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 27418,
  "failing-test-cases": [
    {
      "className": "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest",
      "failingMethods": [
        "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest#testRetryAllFail",
        "io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest#testRetrySuccessAtSecondLocation"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "isError": false
        },
        {
          "failureName": "java.lang.AssertionError",
          "isError": false
        }
      ],
      "nbFailures": 2,
      "nbErrors": 0
    },
    {
      "className": "io.druid.segment.loading.LocalDataSegmentPusherTest",
      "failingMethods": [
        "io.druid.segment.loading.LocalDataSegmentPusherTest#testPushCannotCreateDirectory"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Expected test to throw (an instance of java.io.IOException and exception with message a string containing \"Unable to create directory\")",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "io.druid.query.lookup.LookupSnapshotTakerTest",
      "failingMethods": [
        "io.druid.query.lookup.LookupSnapshotTakerTest#testIOExceptionDuringLookupPersist"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Expected exception: io.druid.java.util.common.ISE",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ],
  "nbCPU": 4
}